src/stream/src/executor/aggregation/mod.rs

```diff
@@ -384,3 +388,24 @@ pub async fn generate_managed_agg_state<S: StateStore>(
         prev_states: None,
     })
 }
+
+/// Get the pk keys len (Do not count group key).
+/// For hash agg, add with group key to get internal table primary key len.
+/// For simple agg,
+pub fn get_key_len(agg_call: &AggCall) -> usize {
```

Comment:
This function is introudced as a temporal solution to get primary key length of state table. Discussed with @st1page offline, we reached agreement that fe should be able to infer it in future and serialize it to prost.